backend/src/Designer/Controllers/PreviewController.cs

```diff
@@ -144,7 +156,7 @@ public IActionResult CurrentUser(string org, string app)
         /// <param name="app">Application identifier which is unique within an organisation.</param>
         /// <returns>An example party</returns>
         [HttpGet]
-        [Route("authorization/parties/current")]
+        [Route("api/authorization/parties/current")]
```

Comment:
Is this right? These endpoints mock the app app endpoints, do they have ‘api’ in the path like this?

Reply:
Just the way it was... I just removed `api` from the base route so that we could use the same controller for the `index.html`